aptos-move/framework/supra-framework/sources/genesis.move

```diff
@@ -617,7 +617,7 @@ module supra_framework::genesis {
         chain_status::set_genesis_end(supra_framework);
     }
 
-    #[verify_only]
+    #[test_only]
```

Comment:
@sjoshisupra Should we also add this for verification? I hope this will not fail move prover.

Reply:
But I am fine with the change, I can check it later.

Reply:
@axiongsupra , the compiler complains if the same directive appears twice, once for `[test_only]` and once for `[verify_only]`.

Reply:
I checked out this branch, with out the verify_only import, the compilation will fail because of unbound module
```
error: unbound module
    ┌─ /Users/aosenxiong/supra/aptos-core/aptos-move/framework/supra-framework/sources/genesis.move:663:9
    │
663 │         features::change_feature_flags(supra_framework, vector[1, 2, 11], vector[]);
    │         ^^^^^^^^ Unbound module alias 'features'
```

Reply:
fixed by the commit that I added.